File: VB/CustomizeNewActionItemsListExample.Module/BusinessObjects/Task.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo
Imports DevExpress.Persistent.Base

Namespace CustomizeNewActionItemsListExample.Module.BusinessObjects
    <DefaultClassOptions> _
    Public Class DemoTask
        Inherits Task

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub
        Public Property Deadline() As Date
    End Class
End Namespace

File: CS/CustomizeNewActionItemsListExample.Module/BusinessObjects/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;
using DevExpress.Persistent.Base;

namespace CustomizeNewActionItemsListExample.Module.BusinessObjects {
    [DefaultClassOptions]
    public class DemoTask : Task {
        public DemoTask(Session session) : base(session) { }
        public DateTime Deadline { get; set; }
    }
}
